pkg/crc/cluster/podman.go

```diff
@@ -7,12 +7,12 @@ import (
 )
 
 func GenerateCockpitBearerToken() string {
-	rand.Seed(time.Now().UnixNano())
+	rng := rand.NewSource(time.Now().UnixNano())
```

Comment:
not strictly needed for us, but `golangci-lint` static check started complaining after updating to `go 1.20`, `rand.Seed` has been deprecated 

```sh
pkg/crc/cluster/podman.go:10:2: SA1019: rand.Seed has been deprecated since Go 1.20 and an alternative has been available since Go 1.0: Programs that call Seed and then expect a specific sequence of results from the global random source (using functions such as Int) can be broken when a dependency changes how much it consumes from the global random source. To avoid such breakages, programs that need a specific result sequence should use NewRand(NewSource(seed)) to obtain a random generator that other packages cannot access. (staticcheck)
        rand.Seed(time.Now().UnixNano())
        ^
```